packages/@okta/i18n/src/properties/login_nb.properties

```diff
@@ -633,7 +636,7 @@ password.complexity.length = minst {0} tegn
 password.complexity.list.element = , {0}
 
 # One or more of the following list elements will be included in the list
-password.complexity.lowercase = en sm\u00E5 bokstav
+password.complexity.lowercase = en liten bokstav
```

Comment:
[https://oktainc.atlassian.net/browse/OKTA-461436](https://oktainc.atlassian.net/browse/OKTA-461436) P1 ticket.

Reply:
Confirming the existing translation is a grammatical number error singular vs. plural mismatch.

Singular: en liten bokstav  (one lowercase letter)
Plural: små bokstaver (lowercase letters)